### DISCUSSION TITLE
My email is hacked, I cannot recover , how can I change my GitHub email to my new email?

### DISCUSSION THREAD

**Discussion by tomipiriyev** [community/community#139367] [Other Feature Feedback, Questions, & Ideas]
Hi , 

My gmail is hacked, and I cannot recover , how can I change my GitHub email to my new email?

thanks in advance

**Comment by davevad93** [comment]
Hi @tomipiriyev ,

Go to your [email settings](https://github.com/settings/emails), add the new email, verify it and set it as your primary email address, you should receive an email from GitHub to verify your new address, when it has been successfully verified you can safely remove your old email from the settings.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @tomipiriyev ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by im-shubh** [comment]
Hey @tomipiriyev 

If your Gmail is hacked and you're unable to recover it, here's how you can change your GitHub email to a new one:

**1. Log into GitHub**  using your existing credentials.

**2. Add a new email:**
->Go to **[Settings > Emails.](https://github.com/settings/emails)** 
->Under **Add email address,** enter your new email and click Add.

**3. Verify your new email:**
GitHub will send a verification link to your new email. Click on the link to verify it.

**4. Set your new email as primary:**
->After verifying the new email, go back to **[Settings > Emails](https://github.com/settings/emails).**
->Next to your new email, click **Make primary**.

**5. Remove your old email:**
Once your new email is set as primary, you can remove the old (hacked) email from your account.

**6. Enable 2FA (Two-Factor Authentication):**
For added security, consider enabling two-factor authentication in **Settings > Account security**.

Let me know if you need further assistance.

Thanks!

**Comment by arslan578** [comment]
To change your GitHub email if you no longer have access to your Gmail account, follow these steps:

1. **Log into GitHub** using your username and password (if you're already logged in, skip this step).

2. Go to **Settings**:
   - Click on your profile picture in the upper-right corner.
   - Select **Settings** from the dropdown menu.

3. **Update your email**:
   - In the left sidebar, click on **Emails**.
   - Add your **new email address** by entering it in the "Add email address" field.
   - Click **Add**.

4. **Verify your new email**:
   - GitHub will send a verification email to your new address.
   - Open the email and follow the verification link.

5. **Set the new email as primary**:
   - Once verified, go back to **Emails** in the settings.
   - Set the new email as your **primary email** by clicking the "Set as primary" button next to your new email.

6. **Remove the old email**:
   - After updating, you can safely remove the hacked Gmail by clicking the **Delete** icon next to it.

This will update your GitHub account with your new email address. ✔️
